src/components/audittable/columns.tsx

```diff
@@ -0,0 +1,97 @@
+"use client"
+
+import { ColumnDef } from "@tanstack/react-table"
+import { MoreHorizontal } from "lucide-react"
+import {Checkbox} from "@/components/ui/checkbox"
+
+import { Button } from "@/components/ui/button"
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu"
+
+
+// This type is used to define the shape of our data.
+export type FetchedCalls = {
```

Comment:
instead of fetchedcalls, can this be renamed so that it is for calls and messages?